app/controllers/concerns/blacklight/bookmarks.rb

```diff
@@ -136,6 +134,10 @@ def clear
 
   protected
 
+  def search_service
+    Blacklight::SearchService.new(blacklight_config)
```

Comment:
I think `BookmarksController < CatalogController`. Do we need to do this explicitly, or can it just be inherited?